metadata-db-app/src/cmr/metadata_db/data/oracle/concepts.clj

```diff
@@ -376,46 +377,58 @@
+                               ()))]
+     (when (not (= "dynamo-off" (dynamo-config/dynamo-toggle)))
        (info "Runtime of EFS get-concept: " (first efs-concept-get) " ms.\nValue gotten from EFS get-concept: " (second efs-concept-get)))
-     (when (not (= "efs-only" (efs-config/efs-toggle)))
+     (when (not (= "dynamo-only" (dynamo-config/dynamo-toggle)))
        (info "Runtime of Oracle get-concept: " (first oracle-concept-get) " ms."))
+     (when (not (= "dynamo-off" (dynamo-config/dynamo-toggle)))
+       (info "Runtime of DynamoDB get-concept: " (first dynamo-concept-get) " ms.\nValue gotten from DynamoDB get-concept: " (second dynamo-concept-get)))
      (if oracle-concept-get
-       (if (not (= "efs-off" (efs-config/efs-toggle)))
+       (if (not (= "dynamo-off" (dynamo-config/dynamo-toggle)))
          (assoc (second oracle-concept-get) :metadata (:metadata (second efs-concept-get)))
          (second oracle-concept-get))
-       (second efs-concept-get))))
+       (second dynamo-concept-get))))
   ([db concept-type provider concept-id revision-id]
    (if revision-id
      (let [table (tables/get-table-name provider concept-type)
-           efs-concept-get (when (not (= "efs-off" (efs-config/efs-toggle)))
+           efs-concept-get (when (not (= "dynamo-off" (dynamo-config/dynamo-toggle)))
                              (util/time-execution (efs/get-concept provider concept-type concept-id revision-id)))
-           oracle-concept-get (when (not (= "efs-only" (efs-config/efs-toggle)))
+           oracle-concept-get (when (not (= "dynamo-only" (dynamo-config/dynamo-toggle)))
                                 (util/time-execution
                                  (j/with-db-transaction
                                    [conn db]
                                    (db-result->concept-map concept-type conn (:provider-id provider)
                                                            (su/find-one conn (select '[*]
                                                                                      (from table)
                                                                                      (where `(and (= :concept-id ~concept-id)
-                                                                                                  (= :revision-id ~revision-id)))))))))]
-       (when (not (= "efs-off" (efs-config/efs-toggle)))
-         (info "Runtime of EFS get-concept: " (first efs-concept-get) " ms.\nValue gotten from EFS get-concept for concept-id " concept-id " and revision-id " revision-id ": " (second efs-concept-get)))
-       (when (not (= "efs-only" (efs-config/efs-toggle)))
+                                                                                                  (= :revision-id ~revision-id)))))))))
+           dynamo-concept-get (when (not (= "dynamo-off" (dynamo-config/dynamo-toggle)))
+                                (util/time-execution
+                                 ()))]
```

Comment:
Is this an extra internal `()`?

Reply:
Thats for the next ticket of actually implementing the functionality.